pkg/models/apiv1.go

```diff
@@ -130,10 +130,15 @@ type ClusterBackup struct {
 
 type BackupDataCentre struct {
 	Nodes []*BackupNode `json:"nodes"`
+	Name  string        `json:"name"`
+	ID    string        `json:"id"`
 }
 
 type BackupNode struct {
-	Events []*BackupEvent `json:"events"`
+	Events         []*BackupEvent `json:"events"`
+	PrivateAddress string         `json:"privateAddress,omitempty"`
+	PublicAddress  string         `json:"publicAddress"`
+	ID             string         `json:"id"`
```

Comment:
Why do we use APIv1 structures when the resource is migrated to APIv2?

Reply:
fixed